### ISSUE TITLE
ASIO

### ISSUE THREAD

**Issue by HerrEmmerich** [w-okada/voice-changer#727]
Cannot use the template because 'cannot submit, green submit button stays greyed out

**Comment by w-okada** [comment]
use issue template

**Comment by w-okada** [comment]
Have you filled out all the necessary fields? If the button is still grayed out, it might be a bug on GitHub. Please contact GitHub for assistance.

**Comment by HerrEmmerich** [comment]
Yes, I think it's a bug on their end. I will get in contact with GitHub. Thank you!
